src/screens/CollectionEditScreen.tsx

```diff
@@ -114,11 +114,24 @@ export default function CollectionEditScreen(props: PropsType) {
       }
 
       await dispatch(collectionUpload(colMgr, collection));
-      dispatch(pushMessage({ message: "Collection saved", severity: "success" }));
-      navigation.goBack();
-      // FIXME having the sync manager here is ugly. We should just deal with these changes centrally.
-      const syncManager = SyncManager.getManager(etebase);
-      dispatch(performSync(syncManager.sync())); // not awaiting on puprose
+      if (colUid || colUid.length > 0) {
```

Comment:
This looks very wrong. First of all, where did this collection come from, but also, why or?

Reply:
I'm surprised typescript didn't complain about this. It definitely should, unless the type of colUid is always defined and not null in the first place (which is bad if is the case).

Reply:
Actually, the first test is unnecessary… Because the colUid is defined as an empty string if route.params.colUid is undefined. So it is never undefined.

Reply:
Cool, just let me know when it's ready for review.